platformio/ide/tpls/clion/CMakeListsPrivate.txt.tpl

```diff
@@ -5,6 +5,8 @@
 # please create `CMakeListsUser.txt` in the root of project.
 # The `CMakeListsUser.txt` will not be overwritten by PlatformIO.
 
+%from platformio.project.helpers import (get_project_dir,load_project_ide_data)

```

Comment:
> get_project_dir

There is exported `project_dir` variable which is already normalized for Windows OS. See 
> https://github.com/platformio/platformio-core/blob/develop/platformio/ide/projectgenerator.py#L73